resources/lang/en/order.php

```diff
@@ -40,9 +40,9 @@
             'quantity'=>'QUANTITY',
             'status'=>'Status: ',
             'subtotal'=>'SUBTOTAL',
-            'total'=>'Total: ',
             'taxes'=>'Taxes',
-
+            'total'=>'Total: ',
+            'visa'=>'VISA ending: **** 1234',
```

Comment:
You should only insert text into the lang section, not special characters or anything that does not need a translation.